src/components/PostEditor.js

```diff
@@ -45,13 +55,26 @@ export default class PostEditor extends Component {
         this.quill.on('selection-change', this.onSelection.bind(this));
     }
 
+    insertLink(url) {
```

Comment:
I think we should pass quill instance to `HyperlinkingWrapper` component and have this function there.